cristin-project/src/test/resources/api_response_non_enriched_projects.json

```diff
@@ -1,11 +1,11 @@
 {
   "@context": "https://bibsysdev.github.io/src/project-search-context.json",
-  "id": "https://api.dev.nva.aws.unit.no/cristin/project/?language=nb&page=1&query=reindeer&results=5",
+  "id": "https://api.dev.nva.aws.unit.no/cristin/project?query=reindeer&language=nb&page=1&results=5",
   "size": 135,
-  "searchString": "language=nb&page=1&query=reindeer&results=5",
+  "searchString": "query=reindeer&language=nb&page=1&results=5",
```

Comment:
Its not alphabetical order? Does that go okay with tests?

Reply:
There now is a mismatch between searchString etc and nextResults previousResults. Dont know why you changed the order? is there some functional requirement?

Reply:
Nope, just to make the  tests pass.. We need something to compare objects other than just comparing  json strings